src/frequenz/repo/config/nox/config.py

```diff
@@ -86,6 +87,13 @@ class Config:
     tools invoked by the sessions.
     """
 
+    root_path: str | None = _dataclasses.field(default_factory=lambda: None)
```

Comment:
```suggestion
    root_path: str | None = None
```

The `default_factory` is only necessary if the type is mutable, for example `list`, because if you just use `[]` then the list gets bound to the class forever (same as with passing arguments).